internal/gcs/protocol.go

```diff
@@ -136,8 +136,10 @@ func (typ msgType) String() string {
 	return s + ")"
 }
 
+// todo: move this to .\internal\oc and unify with host request?
+
```

Comment:
did you mean to remove this?

Reply:
yeah, i created a work item
removed it and the previous todo comment